view/base/templates/imprint/email_footer.phtml

```diff
@@ -8,89 +8,106 @@
  */
 ?>
-<p><strong><?= $block->escapeHtml($block->getShopName()) ?></strong></p>
+<p><strong><?= $block->escapeHtml($block->getImprintValue('shop_name')) ?></strong></p>
 
 <p>
-    <?= $block->escapeHtml($block->getCompanyFirst()) ?><br/>
-    <?php if (strlen(trim($block->getCompanySecond()))): ?>
-        <?= $block->escapeHtml($block->getCompanySecond()) ?><br/>
+    <?= $block->escapeHtml($block->getImprintValue('company_first')) ?><br/>
+    <?php if (strlen(trim($block->getImprintValue('company_second')))): ?>
+        <?= $block->escapeHtml($block->getImprintValue('company_second')) ?><br/>
     <?php endif ?>
-    <?= $block->escapeHtml($block->getStreet()) ?><br/>
-    <?= $block->escapeHtml($block->getZip()) ?> <?= $block->escapeHtml($block->getCity()) ?>
+    <?= $block->escapeHtml($block->getImprintValue('street')) ?><br/>
+    <?= $block->escapeHtml($block->getImprintValue('zip')) ?>
+    <?= $block->escapeHtml($block->getImprintValue('city')) ?>
 </p>
 
 <p>
-    <?php if (strlen(trim($block->getTelephone()))): ?>
-        <?= $block->escapeHtml(__('Telephone')) ?>: <?= $block->escapeHtml($block->getTelephone()) ?>
-        <?php if (strlen(trim($block->getTelephoneAdditional()))): ?>
-            (<?= $block->escapeHtml($block->getTelephoneAdditional()) ?>)
+    <?php if (strlen(trim($block->getImprintValue('telephone')))): ?>
+        <?= $block->escapeHtml(__('Telephone')) ?>:
+        <?= $block->escapeHtml($block->getImprintValue('telephone')) ?>
+        <?php if (strlen(trim($block->getImprintValue('telephone_additional')))): ?>
+            (<?= $block->escapeHtml($block->getImprintValue('telephone_additional')) ?>)
         <?php endif ?>
         <br/>
     <?php endif ?>
-    <?php if (strlen(trim($block->getFax()))): ?>
-        <?= $block->escapeHtml(__('Fax')) ?>: <?= $block->escapeHtml($block->getFax()) ?><br/>
+    <?php if (strlen(trim($block->getImprintValue('fax')))): ?>
+        <?= $block->escapeHtml(__('Fax')) ?>: <?= $block->escapeHtml($block->getImprintValue('fax')) ?><br/>
     <?php endif ?>
-    <?php if (strlen(trim($block->getWeb()))): ?>
+    <?php if (strlen(trim($block->getImprintValue('web')))): ?>
         <?= $block->escapeHtml(__('Web')) ?>:
-        <a href="http://<?= $block->escapeHtmlAttr($block->getWeb()) ?>"
-           title="<?= $block->escapeHtmlAttr($block->getCompanyFirst()) ?>">
-            <?= $block->escapeHtml($block->getWeb()) ?>
+        <a href="https://<?= $block->escapeHtmlAttr($block->getImprintValue('web')) ?>"
+           title="<?= $block->escapeHtmlAttr($block->getImprintValue('company_first')) ?>">
+            <?= $block->escapeHtml($block->getImprintValue('web')) ?>
         </a><br/>
     <?php endif ?>
-    <?= $block->escapeHtml(__('E-Mail')) ?>: <?= /* @noEscape */ $block->getEmail(false) ?>
+    <?php if (strlen(trim($block->getImprintValue('web')))): ?>
+        <?= $block->escapeHtml(__('E-Mail')) ?>: <?= /* @noEscape */ $block->getEmail(false) ?>
```

Comment:
Why are you using a special `getEmail` function here instead of `getImprintValue`?

Reply:
Good question. I guess the answer is... Because it was done like that before and there is a special method for it? :joy: :man_shrugging: Internally, it does the same.